builtin/clone.c

```diff
@@ -990,7 +990,7 @@ int cmd_clone(int argc, const char **argv, const char *prefix)
 	struct transport_ls_refs_options transport_ls_refs_options =
 		TRANSPORT_LS_REFS_OPTIONS_INIT;
 
-	git_config(platform_core_config, NULL);
+	git_config(git_default_core_config, NULL);
```

Comment:
I see this is a `fixup!`, but depends on the previous commit. I assume you'll remember to reorder the commits in the next rebase?

Reply:
Indeed. The individual commit needs to go to the Git mailing list early, I think. Or maybe we should combine it with contributing the symlink support (but that's a bigger task as those patches are entangled with the longPaths and the FSCache topics).